.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+# JavaScript Node CircleCI 2.0 configuration file
+version: 2
+jobs:
+  checkout-and-test:
+    docker:
+      - image: circleci/node:lts
+    ## Uncomment the following lines to enable mongo-db
+    #  - image: circleci/mongo:4.0.1
+    # environment:
+    #   - DB_URL:mongodb://127.0.0.1:27017
+    working_directory: ~/batch-12-WeConnect
+    steps:
+      - checkout
+      - run:
+          name: Update npm
+          command: 'sudo npm install -g npm@latest'
```

Comment:
you don't need to update npm. It comes bundled with node

Reply:
Ok. It was only a precaution.

Reply:
yes, there are no deployment files and it's intentional because the deployments are auto-managed using app integrations.

Reply:
The point is to learn how to do those using CI/CD
Example
- test if linting is correct
- run tests
- build code ( if applicable)
- deploy


Unless all these are present in the circle ci, the file only with installing node modules isn't doing anything
cc @nimish-gupta

Reply:
Ok, i have made all of the suggested changes. @rishabh1403